oviewer/convert_es.go

```diff
@@ -169,193 +181,292 @@ func (es *escapeSequence) convert(st *parseState) bool {
 	return false
 }
 
-// csToStyle returns tcell.Style from the control sequence.
-func csToStyle(style tcell.Style, params string) (tcell.Style, error) {
-	if params == "0" || params == "" || params == ";" {
-		return tcell.StyleDefault.Normal(), nil
+// sgrStyle returns tcell.Style from the SGR control sequence.
+func sgrStyle(style tcell.Style, paramStr string) tcell.Style {
+	if paramStr == "0" || paramStr == "" || paramStr == ";" {
```

Comment:
I would suggest using a switch with one case listing the 3 values

It will improve readability

Reply:
Thank you.